Core/Helper.php

```diff
@@ -368,6 +366,7 @@ public static function getShop()
     }
 
     /**
+<<<<<<< HEAD
```

Comment:
I think this is a leftover from a merge conflict

Reply:
removed 😓